sql/drop.go

```diff
@@ -87,3 +88,67 @@ func (p *planner) DropTable(n *parser.DropTable) (planNode, error) {
 	}
 	return &valuesNode{}, nil
 }
+
+// DropDatabase drops a database.
+func (p *planner) DropDatabase(n *parser.DropDatabase) (planNode, error) {
+	if n.Name == "" {
+		return nil, errEmptyDatabaseName
+	}
+
```

Comment:
@MBerhault if root created database a, then created table a.t, then grant write on database a to testuser. Then testuser try to drop database a, then will encounter error "user testuser does not have WRITE privilege on table a.t". Is this expected?